### PR TITLE
Wrappers WIP

### PR DESCRIPTION
**Pull request by PaLeroy**
Improve wrapper part of the code

### CODE REVIEW

File: imp_wrappers/pettingzoo_env.py
Comment (PaLeroy) [inline imp_wrappers/pettingzoo_env.py:1]: I believe we do not need this file?

File: imp_wrappers/pettingzoo_struct.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from imp_wrappers.pettingzoo_env import ParallelEnv
```
Comment (PaLeroy) [inline imp_wrappers/pettingzoo_struct.py]: I think it is better to have from pettingzoo import Parallel Env and people to install petting zoo in their python environment.

**Review by PaLeroy — COMMENTED** [review]
Other than my comments, you got the point!
With each wrapper, we will add a "test_wrappers" script to execute episodes with random actions.
+ all the config options of the paper are in the pymarl wrappers, maybe we have to add it in our example wrappers?

File: imp_wrappers/pettingzoo_struct.py (hunk 1 above)
Comment (moratodpg) [reply imp_wrappers/pettingzoo_struct.py]: Agreed and template removed :)

**Comment by PaLeroy** [comment]
I changed the organisation, tell me what you think about it.

**Review by PaLeroy — COMMENTED** [review]
@moratodpg Can you check everything is ok for you?

**Pull request merged**